src/components/index-page/IndexOneSmallStep.tsx

```diff
@@ -62,23 +62,31 @@ export const IndexOneSmallStep = () => {
           </p>
           <form className="index-one-small-step__form" onSubmit={handleSubmit(onSubmit)}>
             <input
-              className={`index-one-small-step__input ${errors.name ? "index-one-small-step__input__error" : ""}`}
+              className={`index-one-small-step__input ${
```

Comment:
we should probably create a helper function that checks the errors and injects the right class in here. 

note: I think the css you'd want here is a modifier. so it should be something like `one-small-step__input_error`